src/DependencyInjection/Compiler/BunnyProducerInjectPass.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Skrz\Bundle\BunnyBundle\DependencyInjection\Compiler;
+
+use Skrz\Bundle\BunnyBundle\DependencyInjection\SkrzBunnyExtension;
+use Skrz\Bundle\BunnyBundle\Service\BunnyManager;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class BunnyProducerInjectPass implements CompilerPassInterface
+{
+
+	public function process(ContainerBuilder $container): void
+	{
+		$manager = new Reference(BunnyManager::class);
```

Comment:
Prosím, proč se vytváří `$manager = new Reference(BunnyManager::class);` a ne rovnou `$manager = new BunnyManager()` ? 

Spíš jen neznám třídu `Reference` tak mi to neni jasné, díky

EDIT: jestli jsem to dobře pochopil, tak si to sáhne do containeru a tam hledá třídu???

Reply:
tohle pomaha stavit container symfony a kdyz delas compiller pas tak do definici davas reference na classy a ne classy samotne protoze ty by mohli take mit zavyslosti ktere jsou potreba vyresit